cherrypy/_cpcompat.py

```diff
@@ -332,3 +332,13 @@ def _args_from_interpreter_flags():
             args.append('-W' + opt)
 
         return args
+
+if six.PY2:
+    from cgi import escape as escape_html
+else:
+    from html import escape
+
+    def escape_html(s):
```

Comment:
Yet, I'm still unsure which default should we fall back to.
@jaraco, shouldn't we escape html safely?